#233: Support for live reload (#234)
DiceDB/dice: README.md

$ go run main.go
```

> Enjoy a hussle free hot reload function using [Air package](https://github.com/air-verse/air)

Reviewer: Add this section under subheading "Live Development Server"

> Enjoy a hussle free hot reload function using [Air package](https://github.com/air-verse/air)

Reviewer: ```suggestion
DiceDB provides a hot-reloading development server, which allows you to instantly view your code changes in a live server. This functionality is supported by [Air](https://github.com/air-verse/air)
```

To Install Air on your system you have following options.

1. If you're on go 1.22+

> `go install github.com/air-verse/air@latest`

Reviewer: ```suggestion
```sh
go install github.com/air-verse/air@latest
```
```

> 2. Install using binary

Reviewer: ```suggestion
2. Install the Air binary
```

```sh

> # binary will be $(go env GOPATH)/bin/air

Reviewer: ```suggestion
# binary will be installed at $(go env GOPATH)/bin/air
```

curl -sSfL https://raw.githubusercontent.com/air-verse/air/master/install.sh | sh -s -- -b $(go env GOPATH)/bin
```

> Once `air` is installed you can verify using command `air -v`

Reviewer: ```suggestion
Once `air` is installed you can verify the installation using the command `air -v`
```

> To run the DiceDB for local development:

Reviewer: ```suggestion
To run the live DiceDB server for local development:
```

> ```

Reviewer: ```suggestion
```sh
```

$ go run main.go
```

### Live Development Server

DiceDB provides a hot-reloading development environment, which allows you to instantly view your code changes in a live server. This functionality is supported by [Air](https://github.com/air-verse/air)

To Install Air on your system you have following options.

1. If you're on go 1.22+
```sh
go install github.com/air-verse/air@latest

> 

Reviewer: Add closing backticks
